src/main/java/jenkins/plugins/git/traits/GitBrowserSCMSourceTrait.java

```diff
@@ -110,8 +110,11 @@ public String getDisplayName() {
          */
         @Restricted(NoExternalUse.class) // stapler
         public List<Descriptor<RepositoryBrowser<?>>> getBrowserDescriptors() {
-            return ((GitSCM.DescriptorImpl) Jenkins.getActiveInstance().getDescriptor(GitSCM.class))
-                    .getBrowserDescriptors();
+            GitSCM.DescriptorImpl descriptor = (GitSCM.DescriptorImpl) Jenkins.getActiveInstance().getDescriptor(GitSCM.class);
+            if (descriptor == null) {
+                return java.util.Collections.emptyList(); // Should be unreachable
```

Comment:
BTW can use `getDescriptorByType`, and as of 2.87 can use `ExtensionList.lookupSingleton` to skip the null check.